zkstack_cli/crates/zkstack/src/commands/chain/gateway/create_tx_filterer.rs

```diff
@@ -0,0 +1,124 @@
+use anyhow::Context;
+use ethers::{abi::parse_abi, contract::BaseContract};
+use lazy_static::lazy_static;
+use xshell::Shell;
+use zkstack_cli_common::{
+    config::global_config,
+    forge::{Forge, ForgeScriptArgs},
+    wallets::Wallet,
+};
+use zkstack_cli_config::{
+    forge_interface::{
+        deploy_gateway_tx_filterer::{
+            input::GatewayTxFiltererInput, output::GatewayTxFiltererOutput,
+        },
+        script_params::DEPLOY_GATEWAY_TX_FILTERER,
+    },
+    override_config,
+    traits::{ReadConfig, SaveConfig, SaveConfigWithBasePath},
+    ChainConfig, EcosystemConfig,
+};
+
+use crate::{
+    admin_functions::{set_transaction_filterer, AdminScriptMode},
+    consts::PATH_TO_GATEWAY_OVERRIDE_CONFIG,
+    messages::MSG_CHAIN_NOT_INITIALIZED,
+    utils::forge::{check_the_balance, fill_forge_private_key, WalletOwner},
+};
+
+lazy_static! {
+    static ref DEPLOY_GATEWAY_TX_FILTERER_ABI: BaseContract =
+        BaseContract::from(parse_abi(&["function runWithInputFromFile() public"]).unwrap(),);
+}
+
+pub async fn run(args: ForgeScriptArgs, shell: &Shell) -> anyhow::Result<()> {
+    let chain_name = global_config().chain_name.clone();
+    let ecosystem_config = EcosystemConfig::from_file(shell)?;
+    let chain_config = ecosystem_config
+        .load_chain(chain_name)
+        .context(MSG_CHAIN_NOT_INITIALIZED)?;
+    let l1_url = chain_config.get_secrets_config().await?.l1_rpc_url()?;
+    let mut chain_contracts_config = chain_config.get_contracts_config()?;
+    override_config(
+        shell,
+        ecosystem_config
+            .link_to_code
+            .join(PATH_TO_GATEWAY_OVERRIDE_CONFIG),
+        &chain_config,
+    )?;
```

Comment:
why?

Reply:
I mean, it's the correct line. but it doesn't belong into create_tx_filter. 
we already have it inside `convert_to_gateway`